dashboard/main.tf

```diff
@@ -21,7 +21,7 @@ resource "aws_ecs_task_definition" "dashboard_task" {
   container_definitions = jsonencode([
     {
       name      = "poseidon-earthquake-dashboard-task"
-      image     = "" # TO FILL
+      image     = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c11-poseidon-dashboard:latest"
```

Comment:
What happens if the image changes name or location? Consider using a variable so this can be updated with ease

Reply:
Great idea